src/main/java/appeng/parts/CableBusContainer.java

```diff
@@ -1251,7 +1251,7 @@ public CableBusRenderState getRenderState()
 		return renderState;
 	}
 
-	private FacadeRenderState getFacadeRenderState( EnumFacing side )
+	private FacadeRenderState getFacadeRenderState( IBlockAccess world, BlockPos pos, EnumFacing side )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "world". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1172)